tests/test-env-py38.yml

```diff
@@ -51,4 +54,4 @@ dependencies:
   - pip:
       # dev
       - shed
-      - odc-geo >=0.1.1a2
+      - git+https://github.com/opendatacube/odc-geo
```

Comment:
Should probably have a version and/or commit pin?

Reply:
yeah, it's just for now, once new version is released with current fixes and maybe with cog/xr tools I'll switch to that